researcher_ids/RI_Claims_V1.md

```diff
@@ -279,19 +286,128 @@ interfaces, such as:
 -   May provide a rich set of internationalization/localization features for
     clients to consume.
 
-## Claim Objects
+## RI Claim Embedded Token
 
 Each [RI claim](#researcher-identity-claim-ri-claim) name maps to an array of
-claim JSON objects ([RI Claim Objects](#researcher-identity-claim-object-ri-claim-object))
-within a "ga4gh" root OIDC claim object (see [example](#example-ri-claims)).
+([RI Claim Embedded Tokens](#researcher-identity-claim-embedded-token-ri-claim-embedded-token))
+which are encoded as
+[Embedded Tokens](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-embedded-token)
+within a "ga4gh" root OIDC claim object.
+
+Non-normative example of a set of RI Claim Embedded Tokens, encoded as Embedded
+Token strings, with a RI Claim named "ResearcherStatus":
+
+```
+{
+  "ga4gh": {
+    "ResearcherStatus": [
+      "eyJhbGciOiJI...aaa",
+      "eyJhbGciOiJI...bbb"
+    ]
+  }
+}
+```
 
-## Claim Object Fields
+An [example](#example-ri-claims) of encoding multiple RI Claims in a more
+reader-friendly form is also available.
 
-Fields within a RI Claim Object are:
+### RI Claim Embedded Token Requirements
 
-#### "**value**" [required]
+-   RI Claim Embedded Tokens MUST conform to the
+    [GA4GH AAI Specification Embedded Token format](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#embedded-token-issued-by-broker)
+    as a base64 string.
```

Comment:
Is is not a base64 string, it is a "JWS Compact Serialization" string as defined at https://tools.ietf.org/html/rfc7515#section-7.1 i.e. it is 3 base64 strings joined by dots.

Reply:
Done